wazimap_ng/datasets/migrations/0112_auto_20210527_0541.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 2.2.13 on 2021-05-27 05:41
+
+from django.db import migrations
+from django.db.models import Count
+
+
+def remove_duplicate_groups(apps, schema_editor):
+    Dataset = apps.get_model("datasets", "Dataset")
+    DatasetData = apps.get_model("datasets", "DatasetData")
+
+    # loop over all datasets
+    for dataset in Dataset.objects.all():
+
+        # Get groups for a dataset
+        groups = dataset.group_set.all()
+
+        # loop over groups and check if group with same name exists more
+        # one time
+        for group in groups.values("name").annotate(gc=Count("name")):
+
+            # Filter group by name and order by -created
+            # So we can get latest updated group
+            filtered_group = groups.filter(name=group["name"]).order_by("-modified")
+            last_updated_group = filtered_group.first()
+
+            # if count of groups in greater than one 
+            # Delete all groups of dataset with same name except last updated group
+            if group["gc"] > 1:
+                filtered_group.exclude(id=last_updated_group.id).delete()
+
+            # Fetch list of subindicator from dataset data
+            subindicators = list(
```

Comment:
we do this because of the change in #276 
We want to make sure we're just showing and using the subindicators that belong to this dataset.